client/src/pages/Challenge.tsx

```diff
@@ -33,7 +33,7 @@ const Challenge = () => {
         // console.log(details);
         return;
       })
-      .catch((err) => {
+      .catch((err: Error) => {
```

Comment:
Is adding the Error type here necessary?

Reply:
typescript will be mad at you if you dont